peer/go.mod

```diff
@@ -4,10 +4,18 @@ go 1.11
 
 require (
 	github.com/davecgh/go-spew v1.1.1
+	github.com/decred/dcrd/addrmgr/v2 v2.0.0 // indirect
```

Comment:
This does not appear to be correct.  It's a direct dependency since it's now imported.